### DISCUSSION TITLE
Welcome to metrics-accelerator Discussions!

### DISCUSSION THREAD

**Discussion by damiancalabresi** [mulesoft-catalyst/metrics-toolkit#59] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by damiancalabresi** [comment]
@gozdy @olamiral-mulesoft @melamedjoel @DavoCoder @panizzag 
Just testing the new feature!

**Comment by sbegg** [comment]
I'm responsible for managing our Anypoint capacity across Cloudhub, RTF (and occasionally on-premise standalone). I am looking at this accelerator to help manage capacity optimizations and provisioning over the next couple of years of projects we are bringing into the platform.

**Comment by gangadhargangi** [comment]
is MuleSoft Metrics Toolkit Supports CloudHub 2.0 ?
